Assets/FrostedGlass/Scripts/CommandBufferBlur.cs

```diff
@@ -14,6 +14,7 @@ public class CommandBufferBlur : MonoBehaviour
     CommandBuffer _CommandBuffer = null;
 
     Vector2 _ScreenResolution = Vector2.zero;
+    RenderTextureFormat _BufferRenderTextureFormat;
```

Comment:
Could you change it to :

[SerializedField]
_TextureFormat = = RenderTextureFormat.ARGB32;

Reply:
Out of curiosity, why should this be serialized?

Reply:
This was a mistake.. this should not be exposed